src/makers/win32/appx.js

```diff
@@ -30,36 +30,56 @@ function spawnSdkTool(exe, params) {
   return spawnPromise(findSdkTool(exe), params);
 }
 
+export const isValidPublisher = (() => {
+  // MakeCert supports publisher strings matching the (X.500 distinguished name) regex below
+  // See https://msdn.microsoft.com/en-us/library/windows/apps/br211441.aspx
+  /* eslint-disable max-len */
+  const validPublisherRegex = /(CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")(, ((CN|L|O|OU|E|C|S|STREET|T|G|I|SN|DC|SERIALNUMBER|(OID\.(0|[1-9][0-9]*)(\.(0|[1-9][0-9]*))+))=(([^,+="<>#;])+|".*")))*/;
+  /* eslint-enable max-len */
+
+  return publisher => validPublisherRegex.test(publisher);
+})();
```

Comment:
Sorry to keep going in circles about it, but now that I see it, I wonder if this sort of thing might be better off upstreamed in an exportable function in `electron-windows-store`.